### ISSUE TITLE
public health datahub is down

### ISSUE THREAD

**Issue by dsconnell** [berkeley-dsep-infra/hubs#5958]
### Your Role

Instructor

### Affiliated Course Name

R for Public Health

### Hub URL

publichealth.datahub.berkeley.edu

### URL to the notebook file (.ipynb, .rmd etc..) with solutions

- 

### How to reproduce

1. Go to https://publichealth.datahub.berkeley.edu/
2. See error "Service Unavailable"

### Number of Impacted Students

1

### Severity of the Issue

Critical

### Issue timeline

Noticed 8/15 at 10am

### Snapshots of issue

![Screenshot 2024-08-15 at 10 18 16](https://github.com/user-attachments/assets/cc2cf4f2-0cd3-4184-ad79-0bc593f1cc57)

### Additional comments

We want students to start using it Monday August 19

**Comment by shaneknapp** [comment]
hey @dsconnell we're in the middle of our maintenance window.  things will be back up by EOW.

**Comment by balajialg** [comment]
@dsconnell Last week, There was an announcement informing instructors using Datahub about the maintenance window. We had requested that instructors share this information with their course staff but somehow the communication didn't reach you. To ensure you don't miss future updates, I have added you to the Google group and forwarded the announcement. Apologies for any inconvenience this may have caused.
